go.mod

```diff
@@ -118,6 +117,8 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require go.uber.org/mock v0.4.0
```

Comment:
always baffles me why some editors don't group the dependencies :see_no_evil:

Reply:
fixed that one